card.go

```diff
@@ -46,13 +46,14 @@ func loadBuf(ns *Namespace, b BufD) Buf {
 
 // Card represents an item on a Page, and holds attributes and data for rendering views.
 type Card struct {
-	data map[string]interface{}
+	data             map[string]interface{}
+	nameComponentMap map[string]interface{} // Cache for cards with items, secondary_items or buttons.
 }
 
 const dataPrefix = "~"
 
 func loadCard(ns *Namespace, c CardD) *Card {
-	card := &Card{make(map[string]interface{})}
+	card := &Card{make(map[string]interface{}), nil}
```

Comment:
I'd make this initialization more verbose because it's extended now. But it's more important to follow the code style of the project:
```go
card := &Card{
  data: make(map[string]interface{}),
  nameComponentMap: nil,
}
```

Reply:
Seems like there is no strict style regarding the Struct initialization. I was able to find examples for both: named and anonymous params. Will go with your suggestion.